### DISCUSSION TITLE
Feature request: darken background when popup is open

### DISCUSSION THREAD

**Discussion by jorisvanr** [Clooos/Bubble-Card#285] [Feature requests]
It would be nice if there would be a layer in-between the popup and the main dashboard that fades in when the popup comes up. This would create a bit of a visual difference between the two. Especially when sliders are overlapping the current setup can feel a bit cramped sometimes. 

<img width="445" alt="image" src="https://github.com/Clooos/Bubble-Card/assets/16778969/a10b1a93-784e-4746-b321-27c14e0f7acf">

**Comment by dnestico** [comment]
Or even a blurred background would be nice!

**Comment by Clooos** [comment]
The ability to open a pop-up partially is mainly here to show something behind it like some infos or an additional menu that you always want displayed like on my screenshots. It could be an optional feature but the main problem is that blurring the view involve modifying its CSS and it's not possible (that I know) outside of a theme. The only option would be indeed to add a dark layer with a bit of opacity in front of everything, I will consider that for a future release.

**Comment by ajml45** [comment]
I would love to see this as well but understand if there isnt a great solution.  If you went the layer route as noted above would it be possible to make it so (there is an option) that if you click anywhere outside the pop-up in the layered space it closes the pop-up?

**Comment by SynoUser-NL** [comment]
Another vote for a darker and\or blurry background when a popup is visible.

Great project by the way! Looks really cool.

**Comment by tikel1** [comment]
+1
